heroic-component/src/main/java/com/spotify/heroic/filter/MatchKeyFilter.java

```diff
@@ -28,15 +28,15 @@
 import lombok.EqualsAndHashCode;
 
 @Data
-@EqualsAndHashCode(of = {"OPERATOR", "value"}, doNotUseGetters = true)
+@EqualsAndHashCode(of = {"OPERATOR", "key"}, doNotUseGetters = true)
 public class MatchKeyFilter implements Filter {
```

Comment:
Is there a specific reason for having @JsonTypeName specified for some filters but not all?